client/webserver/site/src/html/forms.tmpl

```diff
@@ -145,6 +145,43 @@
     <span id="appPassSpan">[[[app_pw_reg]]]</span>
     [[[reg_confirm_submit]]]
   </div>
+  <div class="fs16 mt-4">
+    [[[provided_markets]]]
+    <table class="marketstable">
+      <thead>
+        <tr>
+          <th class="text-center">[[[asset_header]]]</th>
+          <th class="text-center">[[[fee_price_header]]]</th>
+          <th class="text-center">[[[fee_confs_header]]]</th>
+          <th class="text-center">[[[wallet_header]]]</th>
```

Comment:
Since `wallet_header` is an empty string, can we just skip the translation business and just have no text for this column header?